openshift/template.yaml

```diff
@@ -8,91 +8,94 @@ metadata:
     description: f8a-hpf-insights
```

Comment:
Actually you should have referred the following template from app-interface
https://github.com/fabric8-analytics/f8a-hpf-insights/blob/master/openshift/template-prod.yaml

Reply:
@arajkumar It is the same template I renamed, template-prod.yaml to template. I don't see any purpose of having old template.yaml file?

Reply:
lgtm